common/dao-api/src/main/java/org/thingsboard/server/dao/event/EventService.java

```diff
@@ -35,6 +35,8 @@ public interface EventService {
 
     List<EventInfo> findLatestEvents(TenantId tenantId, EntityId entityId, EventType eventType, int limit);
 
+    EventInfo findLatestDebugRuleNodeInEvent(TenantId tenantId, EntityId entityId);
```

Comment:
Let's add optional here.
```suggestion
    Optional<EventInfo> findLatestDebugRuleNodeInEvent(TenantId tenantId, EntityId entityId);
```